### ISSUE TITLE
[CoE Starter Kit - QUESTION] Problem with Audit Log Custom Connector "Office 365 Management API"

### ISSUE THREAD

**Issue by PRusakPL** [microsoft/coe-starter-kit#5254]
### What is your question?

When trying to setup Audit Log Custom Connector **"Office 365 Management API"** (first time installation on brand new  I repeat all steps included in documentation https://learn.microsoft.com/en-us/power-platform/guidance/coe/setup-auditlog#set-up-the-custom-connector 
After pressing Update Connector button I receive following error message (details on attached image below):

> "The request failed with error: '{\"error\":{\"code\":\"0x80060888\",\"message\":\"**Edm Object passed should have the options selected.** \"}}'. The correlation Id is 'e5393e80-d0c8-4d7f-b46a-12add5f8905d'."

![image](https://user-images.githubusercontent.com/61137411/231444669-7132d79e-bdfb-4ea1-80e0-3082a20f1c18.png)

Can you please advise what could go wrong? And how I should proceed with Custom Connector setup? Any help appreciated.

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

4.3

### What app or flow are you having the issue with?

Audit Log Custom Connector - "Office 365 Management API"

**Comment by Jenefer-Monroe** [comment]
The default Identity Providor for the custom connector changed to Generic Oauth 2, so likely what happened is that you authenticated to a bad connection. Unfortunately this isnt obvious and clean up isnt that easy. 
![image](https://user-images.githubusercontent.com/89584709/225247140-70c99bb5-5770-428c-b45a-60c3f45e01dd.png)

I've outlined how to clean up from this here: [issue 4961](https://github.com/microsoft/coe-starter-kit/issues/4961)

**Comment by PRusakPL** [comment]
Thank you for quick reply.
This was something different I guess.
When I filled in Security section and immediatelly wanted to "Update connector" it failed.
When I did exactly the same but before clicking "Update connector" I clicked Defnition arrow (went from stage 2 to stage 3) - it allowed me to update connector successfully. 
Maybe this was this or this was just coincidence - maybe earlier sth wrong was with this API endpoint in my region (???).
Nevertheless it's working now.
Thanks @Jenefer-Monroe for your help.
